### PR TITLE
add example and description to device api

### PR DESCRIPTION
**Pull request by mgold1234**
# Description
related to FIXES:-https://issues.redhat.com/browse/THEEDGE-3384

## Type of change

What is it?

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Documentation update
- [ ] Tests update
- [ ] Refactor

### CODE REVIEW

**Review by loadtheaccumulator — DISMISSED** [review]

File: pkg/models/devices_api.go [hunk 1]
```diff
@@ -4,46 +4,46 @@ package models
...
-	Name              string      `json:"Name"`
-	LastSeen          EdgeAPITime `json:"LastSeen"`
-	CurrentHash       string      `json:"CurrentHash,omitempty"`
-	ImageID           uint        `json:"ImageID"`
-	UpdateAvailable   bool        `json:"UpdateAvailable"`
+	UUID              string `json:"UUID"`
+	AvailableHash     string `json:"AvailableHash,omitempty"`
+	RHCClientID       string `json:"RHCClientID"`
+	Connected         bool   `json:"Connected"`
+	Name              string `json:"Name"`
```
Comment (ldjebran) [inline pkg/models/devices_api.go:11]: many fields are missing example and doc string

File: pkg/models/devices_api.go [hunk 2]
```diff
@@ -4,46 +4,46 @@ package models
...
+	LastSeen          string `example:"2023-07-19T08:09:08.084004+00:00"` // Last datetime that device updated
```
Comment (ldjebran) [inline pkg/models/devices_api.go:12]: many fields are missing json representation

**Comment by ldjebran** [comment]
the automation is failing on this 
```text
10:26:18 2023/07/26 08:26:18 Error parsing type definition 'models.DeviceViewAPI': object is unsupported type in example value `2023-07-19T08:09:08.084004+00:00`
10:26:18 2023/07/26 08:26:18 Error parsing type definition 'models.DeviceViewListAPI': object is unsupported type in example value `2023-07-19T08:09:08.084004+00:00`
10:26:18 2023/07/26 08:26:18 ParseComment error in file /src/github.com/RedHatInsights/edge-api/pkg/routes/devices.go :object is unsupported type in example value `2023-07-19T08:09:08.084004+00:00`
```

**Review by de1987 — APPROVED** [review]

**Review by acosferreira — APPROVED** [review]
lgtm

**Pull request merged**